sdk/attestation/azure-security-attestation/test/ut/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ add_compile_definitions(AZURE_TEST_DATA_PATH="${CMAKE_BINARY_DIR}")
 # Export the test folder for recordings access.
 add_compile_definitions(AZURE_TEST_RECORDING_DIR="${CMAKE_CURRENT_LIST_DIR}")
 
+add_compile_definitions(AZURE_TEST_ASSETS_DIR="${CMAKE_CURRENT_LIST_DIR}/../../../")
```

Comment:
This look incredibly fragile. Is there another way of expressing the service root that doesn't require relative navigation?

Reply:
not that i could find,